openc3/lib/openc3/io/json_rpc.rb

```diff
@@ -115,9 +115,9 @@ def as_json(options = nil) #:nodoc:
 
 class Hash
   def as_json(options = nil) #:nodoc:
-    pairs = []
-    self.each { |k, v| pairs << k.to_s; pairs << v.as_json(options) }
-    Hash[*pairs]
```

Comment:
This was overwhelming the stack with the huge amount of items and causing a 'Stack too deep' error.